Author: Tommaso Moroni <moronito@debian.org>
Description: Correct the command-line option documentation printed with --help.

diff -urNad postgresql-autodoc-1.30~/postgresql_autodoc.pl postgresql-autodoc-1.30/postgresql_autodoc.pl
--- postgresql-autodoc-1.30~/postgresql_autodoc.pl	2007-01-02 15:42:55.000000000 +0100
+++ postgresql-autodoc-1.30/postgresql_autodoc.pl	2007-01-28 08:09:28.000000000 +0100
@@ -1844,7 +1844,7 @@
     my ( $basename, $database, $dbuser ) = @_;
     print <<USAGE
 Usage:
-  $basename [options] [dbname [username]]
+  $basename [options]
 
 Options:
   -d <dbname>     Specify database name to connect to (default: $database)
